src/main/java/cpp/logic/commands/AddProjectCommand.java

```diff
@@ -0,0 +1,23 @@
+package cpp.logic.commands;
+
+import cpp.exceptions.IllegalCommandException;
+import cpp.model.ProjectList;
+import cpp.response.Response;
+
+
+public class AddProjectCommand extends Command {
+    private final String projectName;
+
+    public AddProjectCommand(String projectName) {
+        this.projectName = projectName;
+    }
+
+    public String execute(ProjectList projectList) {
+        projectList.addProject(projectName);
+        return Response.addProjectSuccessfully(projectName);
+    }
+
+
+
+
+}
```

Comment:
After looking through all of the newly added command classes, they all seem functional and well-implemented